recipes/datarobot/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2016-2018 DataRobot
```

Comment:
I don't think this file is needed if the license is included in the package (we should be able to pull it from the tarball). If it isn't please follow the steps for [packaging a license](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).